pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -274,7 +285,13 @@ func (c *Cluster) getLabels() map[string]string {
 	}
 }
 
-func (c *Cluster) createKeyring(clusterName, name string) error {
+func (c *Cluster) getDaemonLabels(daemonName string) map[string]string {
+	labels := c.getLabels()
+	labels["name"] = daemonName
```

Comment:
I would prefer to use `app` as the name of the label instead of `name`.
Also what about adding the `a` or `b` instance here (e.g.`instance: a`)?

Reply:
`getLabels()` actually sets a couple of labels including the `app=rook-ceph-mgr`. The `name` was intended to be added as the instance name.

Reply:
Wouldn't `instance` as the label name be more fitting then?